google_photos_takeout_helper/__main__.py

```diff
@@ -109,7 +113,7 @@ def for_all_files_recursive(
       folder_function=lambda fo: True,
       filter_fun=lambda file: True
     ):
-        for file in dir.rglob("*"):
+        for file in tqdm(dir.rglob("*")):
```

Comment:
That's a recursive function, that is also used in many different ways, so that won't work out well

Reply:
For me these uses of tqdm work out okay even though the function is used recursively. The display on my console is not perfect yet but does run without causing any errors in the code. Have you been able to run the code with tqdm in place?